ckan/templates-midnight-blue/snippets/license.html

```diff
@@ -14,37 +14,28 @@
   {% if text_only %}
     {{ license_string(pkg_dict) }}
   {% else %}
-      {% block license_wrapper %}
-        <section class="module module-narrow module-shallow license">
-          <div class="accordion" >
-            <div class="accordion-item">
-              {% block license_title %}
-              <h2 class="accordion-header" id="headingLicense">
-                  <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapsedlicense" aria-expanded="true" aria-controls="collapsedlicense">
-                    {{ _('License') }}
-                  </button>
-              </h2>
+    {% block license_wrapper %}
+      <section class="module module-narrow module-shallow license">
+        <div class="module-group">
+          {% block license_title %}
+          <h2 class="module-heading" id="headingLicense">
+                {{ _('License') }}
+          </h2>
+          {% endblock %}
+          {% block license_content %}
+            <p class="module-content">
+              {% block license_content_inner %}
+                {{ license_string(pkg_dict) }}
+                {% if pkg_dict.isopen %}
+                  <a href="http://opendefinition.org/okd/" aria-label="{{ _('This dataset satisfies the Open Definition.') }}" data-bs-title="{{ _('This dataset satisfies the Open Definition.') }}" data-bs-toggle="tooltip">
+                      <img class="open-data" src="{{ h.url_for_static('/base/images/od_80x15_blue.png') }}" alt="[Open Data]" />
```

Comment:
@aleeexgreeen WDYT about old low-res icons like this? Is it worth making higher resolution or SVG versions of these?

Reply:
This icon like something we could render as page content with a little CSS. That way the text can be translated too.

Reply:
I've added this icon with HTML and CSS, it involved wrapping a couple of `span`s in a `trans` block, but let me know if this  is the direction we want to go in.

Reply:
Ahh, the end of an era. This "low-res icon" was one of the original buttons that were created alongside the [Open Definition](https://opendefinition.org/buttons/) back in 2010-ish?. They fitted right along the Web 1.0 myriad of other buttons that web sites used to display. I might misremember but maybe it was @jwyg who put them together? He is definitely the author of the [infamous picture](https://www.flickr.com/photos/jwyg/4528443760/in/photostream/) of a stack of stickers with this logo that has been used everywhere. If I ever have the time I'd like to investigate how to create a vector version of it and I'll try to get it back into CKAN!
![sure](https://github.com/user-attachments/assets/5acf959f-9f0b-4822-b2ae-206d101f03c0)